koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -270,7 +271,10 @@ def get_expected_nvr(self):
     def format_label(self, label_id):
         """Formats string with user-facing LABEL name and its alternatives"""
 
-        assert label_id in LABEL_NAME_MAP
+        if label_id not in LABEL_NAME_MAP:
+            msg = '"%s" does not contain "%s"', LABEL_NAME_MAP, label_id
```

Comment:
```suggestion
            msg = '"%s" does not contain "%s"' % (LABEL_NAME_MAP, label_id)
```

or just use modern way `"".format()`

Reply:
Right on! I wasn't sure how much of a style change to make...

Reply:
I debated with myself about using `format()`, but thought I should match the existing style... if it's OK with you... I'd *prefer* `format()` :D

Reply:
I'm ok with `format`